markdown/GameGuides/DeadSpace.md

```diff
@@ -55,13 +55,13 @@ Based on [original description by Jim2Point0](https://web.archive.org/web/201410
 
 FOV: Field of View.
 
-Cam Distance: How far back the camera sits from character. The further you go, the smaller he is. This is what the "widescreen fixer" app modifies as well.
+Cam Distance: How far back the camera sits from character. The further you go, the smaller char is. This is what the "widescreen fixer" app modifies as well.
```

Comment:
Please next time, submit changes to different guides in different PRs. Also this change is silly, what's 'char' ?

Reply:
Ok, got it.
Char is short for Character. But for sure 'he' would fit here. 
Thanks.